core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -72,7 +72,6 @@ object ConsumerGroupCommand {
     } catch {
       case e: Throwable =>
         println("Error while executing consumer group command " + e.getMessage)
-        println(Utils.stackTrace(e))
```

Comment:
Minor: maybe we could log this at DEBUG instead? In case there is an unexpected error, users can turn on logging to see it.